code/datums/repositories/unique.dm

```diff
@@ -20,6 +22,9 @@ var/repository/unique/uniqueness_repository = new()
 /datum/uniqueness_generator/proc/Generate()
 	return
 
+/datum/uniqueness_generator/hex/Generate()
+	return "[RANDBYTE]-[RANDBYTE]-[RANDBYTE]-[RANDBYTE]"
```

Comment:
While the probability is exceedingly small that a duplicate will occur I'll note that there's nothing inherent in the system that prevents them.
See `/datum/uniqueness_generator/id_random/Generate()` as an example

Reply:
I don't want to change CHINSKY's code more than necessary. Resolving that problem is outside of the scope of this PR, I feel.

Reply:
Merely making a note

Reply:
Comment rescinded. In that case I agree completely.